docs/site/express-with-lb4-rest-tutorial.md

```diff
@@ -262,6 +272,37 @@ export async function main(options: ApplicationConfig = {}) {
 }
 ```
 
+{% include code-caption.html content="index.js" %}
+
+```js
+const application = require('./dist');
+
+module.exports = application;
+
+if (require.main === module) {
+  // Run the application
+  const config = {
+    rest: {
+      port: +process.env.PORT || 3000,
```

Comment:
an extra `+`?

Reply:
No, `+` prefix converts the string into number.